controllers/NUgus_Controller/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+set(controller_sources NUgus_Controller.cpp)
+
+add_executable(nugus_controller ${controller_sources})
+
+target_link_libraries(nugus_controller PRIVATE webots::webots)
```

Comment:
```suggestion
# Link against the webots target (this will also add any necessary include directories to our target)
target_link_libraries(nugus_controller PRIVATE webots::webots)
```